frontend/src/metabase/visualizations/lib/table_format.js

```diff
@@ -21,8 +21,8 @@ type SingleFormat = {
   type: "single",
   columns: ColumnName[],
   color: Color,
-  operator: "<" | ">" | "<=" | ">=" | "=" | "!=",
-  value: number,
+  operator: "<" | ">" | "<=" | ">=" | "=" | "!=" | "is-null" | "not-null",
```

Comment:
schema_metadata uses an underscore instead of a -. Might want to keep it consistent and use IS_NULL and NOT_NULL

Reply:
Actually those need to be updated since we started normalizing MBQL to the lower-case-with-dashes style. In fact filters with dashes in them aren't appearing the query builder correctly.